project/Http4sPlugin.scala

```diff
@@ -23,6 +24,13 @@ object Http4sPlugin extends AutoPlugin {
   override lazy val projectSettings: Seq[Setting[_]] = Seq(
     scalazVersionRewriter := scalazVersionRewriters.scalazStream_0_8,
 
+    // This is set in ScalazPlugin's projectSettings. Why isn't it effective for
+    // us?
+    unmanagedSourceDirectories in Compile +=
+      (sourceDirectory in Compile).value / scalazSourceDirectory(scalazVersion.value),
+    unmanagedSourceDirectories in Test +=
+      (sourceDirectory in Test).value / scalazSourceDirectory(scalazVersion.value),
```

Comment:
I don't know why this was necessary, because the plugin does this, and we got other values from the plugin.  🤷‍♂️